src/System.Data.SqlClient/src/System/Data/SqlClient/SqlCommand.cs

```diff
@@ -889,13 +889,13 @@ private void BeginExecuteNonQueryInternalReadStage(TaskCompletionSource<object>
             }
         }
 
-        private void VerifyEndExecuteState(Task completionTask, String endMethod)
+        private void VerifyEndExecuteState(Task asyncResult, String endMethod)
```

Comment:
It's strongly typed as a Task.  It's not like it's an IAsyncResult that happens to be a Task and is being cast to one internally.